### DISCUSSION TITLE
Help regarding TS API

### DISCUSSION THREAD

**Discussion by L4B0MB4** [LeonMarcel-HD/StreamDeck-TeamSpeak5#12] [Q&A / Help]
Hey, i know this is not your job but maybe you could help me with finding documentation or hint on how payloads need to look like when sending them over the websocket connection. How did you know you have to use "buttonPress" with the resepective payload ? 
Thanks in advance !

**Comment by LeonMarcel-HD** [comment]
You can find the documentation for the Remote Apps in the `Settings` -> `Remote Apps` -> and beside Global you can find a info button which opens the documentation. 

The hotkeys are explained in the documentation and should be clear to most developers. Whenever I had questions, I asked some of my friends or TeamSpeak staff directly.

You can also check out this Forum thread for further help:
https://community.teamspeak.com/t/teamspeak-remote-apps/23250

For examples you can look at my [app.js](https://github.com/LeonMarcel-HD/StreamDeck-TeamSpeak5/blob/main/src/app.js) or at the implementation for the [OBS Overlay](https://github.com/DerTyp7/ts5-obs-overlay/blob/main/src/Viewer.tsx) made by @DerTyp7

As of now you can only receive data and send hotkeys to the Client. Everything else is not implemented... :c

**Reply by L4B0MB4** [reply]
Hey, thank you very much for the detailed answer. Unfortunately i have found all these links already and was hoping im just too stupid to find proper documentation for the feature. But it seems not to be present. 
Anyway, thank you very much !
